roles/update-access/test/test.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
A good test would be one that updated the key and actually used it for a following task to see that the key actually works. Same for the password.